src/shared/components/mutationTable/MutationTable.tsx

```diff
@@ -679,7 +679,9 @@ export default class MutationTable<
 
         this._columns[MutationTableColumnType.PROTEIN_CHANGE] = {
             name: 'Protein Change',
-            render: ProteinChangeColumnFormatter.renderWithMutationStatus,
+            render: ProteinChangeColumnFormatter.renderWithMutationStatus(
```

Comment:
I personally find functions which return functions pretty confusing. We should probably do something similar to AnnotationColumnFormatter and explicitly pass whatever needed from the store, instead of the entire store.

https://github.com/cBioPortal/cbioportal-frontend/blob/5774102d1f50bedbf93f9b6da3fec6b4d2d3dc56/src/shared/components/mutationTable/MutationTable.tsx#L791-L809